clients/java/src/main/java/io/zeebe/client/api/JsonMapper.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.client.api;
+
+import java.io.InputStream;
+import java.util.Map;
+
+/**
+ * This interface is using to customize the way how objects will be transformed in JSON format.
+ *
+ * @see io.zeebe.client.impl.ZeebeObjectMapper
+ */
+public interface JsonMapper {
+
+  /**
+   * Transform a JSON string to the typed object.
```

Comment:
```suggestion
   * Deserializes a JSON string into an equivalent POJO of type {@code T}.
```

I think this is more indicative of the intent, i.e. deserialization.

Reply:
Done